src/b-st-ext.adoc

```diff
@@ -2569,13 +2569,13 @@ Included in::
 
 <<<
 [#insns-revb,reftext="Reverse bits in bytes"]
```

Comment:
Should the name be fixed in `#insns-revb` too?

Reply:
Since that conflicts with a similar tag in scalar-crypto, I'm going to leave it alone for now, but someone should definitely come along and fix that up.